.changeset/fine-places-search.md

```diff
@@ -0,0 +1,5 @@
+---
+"@launchpad-ui/components": minor
+---
+
+export tree component
```

Comment:
for future reference, you can use the `skip changeset` label on the PR + not include this changeset